Set up autopublication on TR

dontcallmedom opened the pull request:
Tweak document and spec-prod set up for automatic publish.

Marking the PR as draft since it shouldn't be merged at least until the first note has been published - see also https://github.com/w3c/transitions/issues/437


<!--
    This comment and the below content is programmatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/webmachinelearning/ethical-webmachinelearning/pull/23.html" title="Last updated on Jun 27, 2022, 11:40 AM UTC (aa6d5f1)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/webmachinelearning/ethical-webmachinelearning/23/a023c87...aa6d5f1.html" title="Last updated on Jun 27, 2022, 11:40 AM UTC (aa6d5f1)">Diff</a>

dontcallmedom committed:
Tweak for publication on TR

Use more neutral shortname
Move status considerations from Abstract to SOTD
Align with Note requirements in SOTD

dontcallmedom committed:
Configure to autopublish with Echidna

dontcallmedom committed:
Tweak into pubrules compliance

dontcallmedom committed:
Fix broken link

anssiko commented:
Thanks, LGTM!

I'd propose we rename this repo to match the TR shortname `webmachinelearning-ethics`. I'm a fan of not breaking any URLs so here's a redirect that works with anchors too: https://github.com/webmachinelearning/webmachinelearning.github.io/pull/29

To coordinate, if you @dontcallmedom agree please feel free to rename this repo ahead this publication and update the `index.bs` accordingly and also merge the above redirection PR.

dontcallmedom committed:
Align TR and github shortnames

per https://github.com/webmachinelearning/ethical-webmachinelearning/pull/23#issuecomment-1167224856
